sdk/identity/identity/BREAKING_CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Breaking Changes
 
+## 4.1.0
+
+As of `@azure/identity` 4.1.0, the number of IMDS probing retries has been increased to 5 (from 3 initially) to match the [IMDS retry guidance](https://learn.microsoft.com/entra/identity/managed-identities-azure-resources/how-to-use-vm-token#retry-guidance). The users should be able to override the behavior, if required, by setting the `options.retryOptions.maxRetries` in the respective credential.
```

Comment:
Does this breaking change only impact `DefaultAzureCredential` and `ManagedIdentityCredential`? We should explicitly state which credential types are affected.

Reply:
Yes both credentials are affected